multiple-servers/api/images/images.go

```diff
@@ -0,0 +1,48 @@
+package images
+
+import (
+	"context"
+	. "multiple-servers/api/types"
+
+	"github.com/jackc/pgx/v4"
+)
+
+func GetAll(conn *pgx.Conn) ([]Image, error) {
+	rows, err := conn.Query(context.Background(), "SELECT title, alt_text, url FROM images")
+	if err != nil {
+		return nil, err
+	}
+
+	var images []Image
+
+	for rows.Next() {
+		var image Image
+		err = rows.Scan(&image.Title, &image.AltText, &image.Url)
+		if err != nil {
+			return images, err
```

Comment:
I think it highly depends on the service you are offering. 

I wouldn't want to return partial data if it was critial data such as financial information or data that can have many dependencies and cause a chain reaction. 

The reason I went with this approach is that it gives a clear indicator of whether the service is working or not, there's no in-between. I have come accross scearios wher no matter of the request, the response has a standard layout for `data` and `error` keys. This is quite useful, as it will depend on how the end user decides to use that data, e.g. show what's received but notify of error or hard fail and don't show anything.